userspace/libscap/engine/gvisor/scap_gvisor.cpp

```diff
@@ -311,7 +386,7 @@ int32_t engine::process_message_from_fd(int fd)
 
 int32_t engine::next(scap_evt **pevent, uint16_t *pcpuid)
 {
-	struct epoll_event evts[max_ready_sandboxes];
+	epoll_event evts[max_ready_sandboxes];
```

Comment:
Do we have a typedef for it somewhere?

Reply:
The struct prefix before using a struct is required only in C, C++ does not have this constraint

Reply:
Wow didn't know this! 
C++-- .